### DISCUSSION TITLE
Redirecting scripts

### DISCUSSION THREAD

**Discussion by oleedd** [babel/babel#15311] [Q&A]
Can I redirect scripts to compile them? I need something like:
https://babeljs.io/index.php?http://example.com/script.js

**Comment by JLHwung** [comment]
We don't have such plan to support this feature. You can run your own server by calling `transformSync` from `@babel/core` in the route handler.

**Comment by oleedd** [comment]
I don't have a server.
Maybe someone knows a site for this? Maybe not Babel but something similar.
